apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -410,7 +410,8 @@ export class Utilities {
     if (options.handleOutput) {
       Utilities._processResult(result);
     }
-    return result.status;
+
+    return result.status || 1; // If result.status is null, the process was terminated from a signal
```

Comment:
build.is may have the same issue, but it's written without a compiler

Reply:
I don't see `.status` referenced anywhere in any `.js` files except for `common/scripts/install-run.js`

Reply:
The only `.js` file that contains `.status` is `common/scripts/install-run.js`.